skimage/segmentation/__init__.py

```diff
@@ -1,4 +1,4 @@
-"""Algorithms to partition images into meaningful regions or boundaries."""
+"""Partitioning images into meaningful regions or boundaries."""
```

Comment:
I would revert to previous wording, or use "Partition images into meaningful regions or boundaries."

Reply:
My favourite style is "Partition ..." (imperative mood) as in commit messages and function docstrings.

But, since all subpackages have a noun phrase (except for `io` with "Reading and saving of ..."), I agree it would be more consistent to keep "Algorithms to partition ..."

Reply:
Reverted in https://github.com/scikit-image/scikit-image/pull/6892/commits/f69ba1cbd53342d5c2221c1fc9915e634eca5e8c.